libs/wingc/src/type_check/symbol_env.rs

```diff
@@ -290,6 +312,67 @@ impl SymbolEnv {
 		Ok(t)
 	}
 
+	/// Pass `include_hidden: true` if it's OK to return types that have only been imported implicitly (such as through an inheritance chain), and false otherwise
```

Comment:
Maybe a TODO about the code duplication here with `lookup_nested` or perhaps just find a way to remove the code duplication.

Reply:
My rust-fu isn't good enough... a TODO comment will have to do